tests/main/upgrade-from-2.15/task.yaml

```diff
@@ -33,8 +33,30 @@ execute: |
     fi
     wait_for_service snap.go-example-webserver.webserver.service
 
+    echo "Install a test snap"
+    snap install hello-world
```

Comment:
(nitpick) Hey, thanks for adding this smoke test! Is there a downside in using `test-snapd-tools` here? It's slightly more common for this kind of test. OTOH now that the store does not count installs from integration tests for the metrics it probably does not matter I much guess(?).

Reply:
Yes, makes sense, thanks for pushing a change.